tests/create_delete_topic.rs

```diff
@@ -0,0 +1,82 @@
+mod testsupport;
+
+use samsa::prelude::{self, protocol, BrokerConnection, Error, KafkaCode};
+use std::collections::HashMap;
+
+const CLIENT_ID: &str = "create delete topic integration test";
+const CORRELATION_ID: i32 = 1;
+
+#[tokio::test]
+async fn it_can_create_and_delete_topics() -> Result<(), Box<Error>> {
+    let (skip, brokers, _) = testsupport::get_brokers_and_topic()?;
+    if skip {
+        return Ok(());
+    }
+    let conn = BrokerConnection::new(brokers).await?;
+
+    //
+    // Create topic
+    //
+    let mut create_req =
+        protocol::CreateTopicsRequest::new(CORRELATION_ID, CLIENT_ID, 4000, false)?;
```

Comment:
A nice future enhancement would be to avoid these fixed-column `new()` factories by introducing a `CreateTopicsRequestBuilder`.